static/js/components/CourseList.js

```diff
@@ -45,6 +53,18 @@ class CourseList extends React.Component {
         </ul>
         {table}
       </div>;
+
+      let applyForMastersUI = <div className="clear-float">
```

Comment:
yeah, I there's no need to separate it out since we're always including it anyway. if we were conditionally including e.g. the `programListingUI` JSX above it would make sense to me to separate it out, but as-is I think it decreases readability.